raiden-ts/package.json

```diff
@@ -82,6 +82,7 @@
     "io-ts": "^2.1.0",
     "isomorphic-fetch": "^2.2.1",
     "lodash": "^4.17.15",
+    "loglevel": "^1.6.7",
```

Comment:
I tried to find out whether loglevel supports to log to other streams as well but seems like it's only wrapping around `console.log()` right?

Absolutely not in scope of ticket, but maybe a nice option for the future: 
In order to get higher quality bug reports it would be nice to have a way to log to memory per session as well. If then an error occurs, we could show in the frontend the debug output to the user. The user can then go ahead and copy/paste it into github, without us having to ask for the logs all the time

Reply:
Yes, I agree, that would be nice. `loglevel` seems to support plugins and to be very extensive, they even have [a plugin to sending logs to log server](https://github.com/artemyarulin/loglevel-serverSend), but this per-test logging or so would need to be implemented. We may want to do that eventually, thank you